schema.sql

```diff
@@ -31,4 +31,30 @@ ALTER TABLE animals DROP COLUMN species;
 
 ALTER TABLE animals ADD species_id integer, ADD FOREIGN KEY (species_id) REFERENCES species(id);
 
-ALTER TABLE animals ADD owner_id integer, ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
\ No newline at end of file
+ALTER TABLE animals ADD owner_id integer, ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
+
+-- Day4
+
+CREATE TABLE vets (
+    id integer PRIMARY KEY,
```

Comment:
- [ ] Following the project requirements, the id should be autoincremented PRIMARY KEY. You may check out [this link](https://chartio.com/resources/tutorials/how-to-define-an-auto-increment-primary-key-in-postgresql/) to learn how to do that.